services/core-web/src/components/Forms/noticeOfWork/NOWReviewForm.js

```diff
@@ -62,43 +73,140 @@ export class NOWReviewForm extends Component {
   };
 
   render() {
+    const filteredDropDownOptions = this.props.documentTypeOptions.filter(({ subType, value }) => {
+      if (this.props.type === PUBLIC_COMMENT) {
+        return this.props.categoriesToShow.includes(subType) && value !== ADVERTISEMENT_DOC;
+      }
+      return this.props.categoriesToShow.includes(subType);
+    });
+
     return (
       <Form layout="vertical" onSubmit={this.props.handleSubmit}>
         <Row gutter={16}>
           <Col span={24}>
+            {this.props.type === REFERRAL_CODE && (
+              <>
+                <Form.Item>
+                  <Field
+                    id="referral_number"
+                    name="referral_number"
+                    label="E-Referral Number*"
+                    component={renderConfig.FIELD}
+                    validate={[required, maxLength(16)]}
+                  />
+                </Form.Item>
+                <Form.Item>
+                  <Field
+                    id="response_date"
+                    name="response_date"
+                    label="Date Received*"
+                    component={renderConfig.DATE}
+                    validate={[required, dateNotInFuture]}
+                  />
+                </Form.Item>
+              </>
+            )}
+
+            {this.props.type === CONSULTATION_TAB_CODE && (
+              <>
+                <Form.Item>
+                  <Field
+                    id="response_url"
+                    name="response_url"
+                    label="Link to CRTS"
+                    component={renderConfig.FIELD}
```

Comment:
This field and the other ones for consultation aren't _required_?

Reply:
Going to leave them as-is for now until we get more clarification

Reply:
removed the due_date - nothing is required now unless they upload a document, and then the category is required.